src/ext/java/org/rutebanken/tiamat/ext/fintraffic/auth/model/OAuth2ClientToken.java

```diff
@@ -0,0 +1,37 @@
+package org.rutebanken.tiamat.ext.fintraffic.auth.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDateTime;
+
+public class OAuth2ClientToken {
```

Comment:
Where is this class used?

Reply:
Leftover from an interim version, shouldn't be there, will be deleted.